From 6ac50fbfd9016278ed99ba0e1cf186404a2e2ee7 Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Wed, 18 Aug 2021 16:58:11 -0400
Subject: [PATCH] If target_arch not in [x86_64, aarch64], fallback

---
 src/deserialize/deserializer.rs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/deserialize/deserializer.rs b/src/deserialize/deserializer.rs
index c681de3..1816e95 100644
--- a/src/deserialize/deserializer.rs
+++ b/src/deserialize/deserializer.rs
@@ -34,7 +34,7 @@ fn is_valid_utf8(buf: &[u8]) -> bool {
 
 #[cfg(not(any(target_arch = "x86_64", target_arch = "aarch64")))]
 fn is_valid_utf8(buf: &[u8]) -> bool {
-    simdutf8::basic::from_utf8(buf).is_ok()
+    encoding_rs::Encoding::utf8_valid_up_to(buf) == buf.len()
 }
 
 pub fn deserialize(
-- 
2.31.1

