From aabc9f3887fb46f3ec1c78f14047ac6e1edbbdd4 Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Wed, 18 Aug 2021 18:16:12 -0400
Subject: [PATCH 2/2] Specify aarch64_neon feature for simdutf8

---
 Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index 44d0055..78b4645 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -73,7 +73,7 @@ pyo3 = { version = "^0.14.2", default_features = false, features = ["extension-m
 ryu = { version = "1", default_features = false }
 serde = { version = "1", default_features = false }
 serde_json = { version = "^1.0.66", default_features = false, features = ["std", "float_roundtrip"] }
-simdutf8 = { version = "0.1", default_features = false, optional = true }
+simdutf8 = { version = "^0.1.2", default_features = false, optional = true, features = ["aarch64_neon"] }
 smallvec = { version = "^1.6", default_features = false, features = ["union", "write"] }
 
 [profile.release]
-- 
2.31.1

